lib/address_geocoder.rb

```diff
@@ -56,7 +56,7 @@ def call_google
         if attempts <= 5
           retry
         else
-          raise ConnectionError, 'Could not connect to GoogleAPI'
+          raise SystemCallError, 'Could not connect to GoogleAPI'
```

Comment:
:thought_balloon:  i am not sure if SystemCallError suit to the case.
maybe we could refer the following to 
http://stackoverflow.com/questions/5370697/what-s-the-best-way-to-handle-exceptions-from-nethttp#answer-5370726
OR create our own error
https://github.com/rails/rails/blob/7f18ea14c893cb5c9f04d4fda9661126758332b5/activerecord/lib/active_record/errors.rb